README.md

```diff
@@ -1,13 +1,6 @@
-<pre align="center" style="text-align:center; font-size: 1vw; background:none;">
-    __            __        ______
-    /""\          /""\      /    " \
-    /    \        /    \    // ____  \
-    /' /\  \      /' /\  \  /  /    )  )
-  //  __'  \    //  __'  \(: (____/ //
- /   /  \\  \  /   /  \\  \\         \
- (___/    \___)(___/    \___)\"____/\__\
-ASK-A-QUESTION
-</pre>
+<p align="center">
```

Comment:
The full logo does not show up in light mode. 

<img width="825" alt="image" src="https://github.com/IDinsight/aaq-core/assets/14125957/d26962fe-fae5-402a-b98a-a2b69b99059a">

Reply:
Yes this one is an issue. I tried to set a colored background but it was looking ugly. I'll check with Lenny what he suggests

Reply:
Is there a way to switch the image based on what mode it's in?

Reply:
Ah yes, found a fix. Will raise a new PR